src/components/dashboard/forms/web3/SeAddressPrivilege.js

```diff
@@ -45,9 +50,11 @@ class SeAddressPrivilege extends Component {
 		this.props.validate('withdrawAmount', { isValid: isValid, err: { msg: msg, args: errMsgArgs }, dirty: dirty })
 	}
 
-	validateAddress = (addr, dirty) => {
-		const isValid = isEthAddress(addr)
-		const msg = 'ERR_INVALID_ETH_ADDRESS'
+	validateAddress = async (addr, dirty) => {
```

Comment:
You can make function validate `validEthAddress` in the validator helper with bool props e.g. `nonZeroAddr` and `nonERC20`  that returns `error` obj as `Joi` does.

Reply:
Joi just extends Error a little. [Is this what you mean](https://github.com/hapijs/joi/blob/368cef4bb8e01f4b73dfa0bb8e9ddc755c0e5744/lib/errors.js#L223)? For this we should create an error object and use it around the project.
```
exports.ValidationError = class extends Error {

    constructor(message, details, original) {

        super(message);
        this._original = original;
        this.details = details;
    }
```

Reply:
it will be great if we use it like this but now it's not the time. I's Ok with the msg you did it (as some other validation funcs). Just wanted to return object with `error` prop instead the `msg` [as joi](https://hapi.dev/family/joi/?v=15.1.0#validatevalue-schema-options-callback).
Keep it as itis.